capstone/src/main/webapp/js/business.js

```diff
@@ -13,23 +13,53 @@
 // limitations under the License.
 
 import { loadCommentSection } from '/js/comments.js';
+import { setLoginOrLogoutUrl, setProfileUrl } from '/js/util.js';
 
 window.onload = function() {
   // window.location.search returns the search string of the URL.
   // In this case, window.location.search = ?id={businessID}
   const url = new URLSearchParams(window.location.search);
   const businessId = url.get('id');
+  setLoginOrLogoutUrl();
+  setProfileUrl();
   constructBusinessProfile(businessId);
   loadCommentSection(document.getElementById('comment-section'));
 }
 
+// Set the correct values for both view and edit sections.
 function constructBusinessProfile(id) {
-  const profileInfo = document.getElementById('view-profile-section');
+  const profileInfo = document.getElementById('view-business-section');
   fetch('/business/' + id).then(response => response.json()).then(info => {
-    document.getElementById("profile-name").innerText = info.name;
-    document.getElementById("profile-location").innerText = info.location;
-    document.getElementById("profile-bio").innerText = info.bio;
-    // Opens a draft of the email to the business.
-    document.getElementById("profile-email").href = 'mailto:' + info.email;
+    if (info.isCurrentUser) {
+      document.getElementById('edit-button').style.display = 'block';
+    } else {
+      document.getElementById('edit-button').style.display = 'none';
+    }
+
```

Comment:
```suggestion
['name', 'location', 'story', 'bio', 'about', 'support'].forEach({feature =>
    const div = document.getElementById(feature + '-section');
    const label = div.getElementByClass('label');
    label.innerText = info[feature];
    const input = div.getElementByClass('input');
    input.value = info[feature];});
    
```
I know this code won't run but what about doing this something like this so that you don't have to repeat yourself?

Reply:
I added this in if you would like to look at it.